backend/app.py

```diff
@@ -333,6 +346,18 @@ def transactions():
     return dataframe_as_response(df, app.current_request.headers.get('accept'))
 
 
+@app.route('/transactions/{txid}', methods=['DELETE'], cors=CORS_CONFIG,
+           authorizer=get_authorizer())
+def delete_transactions(txid):
+    uuid = get_current_user_email()
+    try:
+        dynamodb.delete_item(**query_by_uuid_and_txid_param(uuid, txid))
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'ConditionalCheckFailedException':
+            return Response(body=f'Invalid transaction id: {txid}',
+                            status_code=400)
+    return Response(body=f'Deleted {txid}', status_code=200)
```

Comment:
return a single `txid` (without `Delete`) might be easier for me to sync the state

Reply:
This endpoint handles a single transaction at a time so when it's 200 it means the transaction is deleted, when it's not it means deletion failed. The response body is for human readability only.